pom.xml

```
@@ -3,7 +3,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>jetbrains.buildServer.clouds</groupId>
     <artifactId>cloud-openstack</artifactId>
-    <version>1.3-SNAPSHOT</version>
+    <version>1.3</version>
```

Development version (-SNAPSHOT) should be kept, release will done on dedicated process.

```suggestion
        <version>1.3-SNAPSHOT</version>
```